lib/cube_browser/__init__.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+
+
+class Contourf(object):
+    def __init__(self, cube, coords, **kwargs):
+        self.cube = cube
+        self.coords = coords # coords to plot
+        self.kwargs = kwargs
+        self.axes = self.new_axes()
+        self.qcs = None  # QuadContourSet
+        # A mapping of 1d-coord name to dimension
+        self.coord_dim = self.coord_dims()
+
+    # This constructs the original and updated plots
+    def __call__(self, **kwargs):
```

Comment:
The choice we take here might be influenced by our approach when we come to deal with anonymous dimensions that are controlled by sliders.

So if we have ...
```python
cf = Contourf(cube, coords=('longitude', 'latitude'))
```
The current approach allows a call with ...
```python
cf(time=10, model_level_number=23)
```
as opposed to something like ...
```python
cf([('time', 10), ('model_level_number', 23)])
```
or ...
```python
cf({'time': 10, 'model_level_number':23})
```
or something else ... I think we need to team-talk this one.